server/database/products.js

```diff
@@ -18,6 +22,7 @@ const checkConnection = (request, response) => {
 }
 
 const getProducts = (request, response) => {
+	console.log(process.env.DATABASE_NAME)
```

Comment:
This kind of logging is useful for debugging. But you should try to avoid leaving them in commits.

Reply:
Thanks! I'll be more careful when using console.logs for debugging